pkg/create/create.go

```diff
@@ -22,11 +22,11 @@ func New(name, dst string) *Bootstrap {
 // AddSTIScripts creates the STI scripts directory structure and process
 // templates for STI scripts
 func (b *Bootstrap) AddSTIScripts() {
-	os.MkdirAll(b.DestinationDir+"/"+".sti/bin", 0700)
-	b.process(templates.AssembleScript, ".sti/bin/assemble", 0755)
-	b.process(templates.RunScript, ".sti/bin/run", 0755)
-	b.process(templates.UsageScript, ".sti/bin/usage", 0755)
-	b.process(templates.SaveArtifactsScript, ".sti/bin/save-artifacts", 0755)
+	os.MkdirAll(b.DestinationDir+"/"+".s2i/bin", 0700)
+	b.process(templates.AssembleScript, ".s2i/bin/assemble", 0755)
+	b.process(templates.RunScript, ".s2i/bin/run", 0755)
+	b.process(templates.UsageScript, ".s2i/bin/usage", 0755)
+	b.process(templates.SaveArtifactsScript, ".s2i/bin/save-artifacts", 0755)
```

Comment:
did you mean to change these?  i wouldn't be totally against changing this path (though we'd have to add backwards compatibility) but it would be a bit painful.

I didn't notice if your edits changed this path in the docs too.